docs/lib/predictions/fragments/ios/identify-entity.md

```diff
@@ -33,6 +33,26 @@ Run `amplify push` to create the resources in the cloud
 
 ## Working with the API
 
+In order to match entities from a pre-created [Amazon Rekognition Collection](https://docs.aws.amazon.com/rekognition/latest/dg/collections.html), make sure these two properties,`collectionId` and `maxEntities`, are set in your `amplifyconfiguration.json` file. `collectionId` should be the  name of your collection that you created either with the CLI or the SDK and `maxEntities` should be a number greater than `0` or less than or equal to `50` which is the max number of entities Rekognition can detect from a collection. If there is no `collectionId` set in the `amplifyconfiguration.json` file or `maxEntities` is set to be `0`, then this call will just detect entities in general with facial features, landmarks, etc. Bounding boxes for entities are returned as ratios so make sure if you would like to place the bounding box of your entity on an image that you multiple the x by the width of the image, the y by the height of the image, and both height and width ratios by the image's respective height and width.
```

Comment:
* Instead of "make sure these two properties...", maybe use: "ensure that both collectionId and maxEntities are set in your amplify... file"
* The value of `collectionId` should be the name of the ..
* separate sentence between explaining the values for `collectionId` and `maxEntities` (it's currently a run on sentence)
* "should be a number greater than.." .. replace with: "should be a number greater than 0 and less than 51."
* change "If there is no `collectionId` set in the `amplifyconfiguration.json` file or `maxEntities` is set to be `0`" to "If both collectionId and maxEntities do not have valid values in `amplifyconfiguration.json` then, this call will instead detect entities..
* I think we should remove info about bounding box... and find a common place for it.  I opened a ticket for it here:
  * https://github.com/aws-amplify/docs/issues/1489

Reply:
k fixed all